<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>604</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Workspace Flow 1.1</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>WORKSPACE.png</normaloff>WORKSPACE.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>971</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="add">
     <attribute name="title">
      <string>Add Hiding Place</string>
     </attribute>
     <widget class="QLineEdit" name="HPName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>871</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>1551</width>
        <height>31</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string>Hiding Place Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>1551</width>
        <height>31</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string>Hiding Place Image Link(optional)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="HPLink">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>871</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_abc">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>1551</width>
        <height>31</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string>Description of Hiding Place</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="Description">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>871</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="savedb_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>450</y>
        <width>871</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit To Online Database</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="view">
     <attribute name="title">
      <string>View Hiding Places</string>
     </attribute>
     <widget class="QTableWidget" name="viewHP">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>921</width>
        <height>441</height>
       </rect>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>Hiding Place Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Hiding Place Image Link</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="RefreshButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>221</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh Hiding Places</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="lcw_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>891</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Live Chat Website(comming in Workspace Flow 2.1)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
